mobile/lib/shared/providers/asset.provider.dart

```diff
@@ -57,7 +57,7 @@ class AssetNotifier extends StateNotifier<AssetsState> {
       debugPrint("newRemote: $newRemote, newLocal: $newLocal");
       await _userService.refreshUsers();
       final List<User> partners =
-          await _db.users.filter().isPartnerSharedByEqualTo(true).findAll();
+          await _db.users.filter().isPartnerSharedWithEqualTo(true).findAll();
```

Comment:
We can make the two properties' names more explicit, for example:

- `partnersThatIShareWith`
- `partnersThatSharedWithMe`

Reply:
Yeah, I've used the naming from the server code to be consistent. Otherwise it's even worse if stuff is named differently.

I can also the change the naming in the server codebase.

Reply:
Then it is ok, you can leave it as is :)

Reply:
FYI - In the partner database table there are two columns (1) shared by and (2) shared with. That means User A _shared_ their library with User B. Direction here in the api request means either (1) shared by (me) or (2) shared with (me).